1/numberDivisors.js

```diff
@@ -0,0 +1,26 @@
+// complexity is O(log n) - I'm not sure
+function numberDivisors(number) {
+  const result = [];
+  for (let divisor = 1; divisor < (number/divisor + 1); divisor++) {
+    if (number % divisor === 0) { 
+      result.push(divisor);
+      divisor === number/divisor? {} : result.push(number/divisor);
+    }
+  }
+  return result;
+}
+
+
+console.log(numberDivisors(6)); // [1,6,2,3]
+console.log(numberDivisors(49)); // [ 1, 49, 7, 7]
+console.log(numberDivisors(1)); // [1, 1] a side effect for 1
```

Comment:
```suggestion
console.log(numberDivisors(49)); // [ 1, 49, 7]
console.log(numberDivisors(1)); // [ 1 ]
```

I guess it works properly now :)